src/draft/Key/LockAssignmentExists.py

```diff
@@ -35,7 +35,8 @@ class LockAssignmentExistsResponse(CamelCaseModel):
 
 
 DEFINITION = DataProductDefinition(
+    title="Check if lock assignment exists",
+    description="Check if lock assignment exists",
```

Comment:
This could maybe be improved to:
```suggestion
    description="Check if a key has access to a specific lock",
```